src/taxonomy/NcbiTaxonomy.h

```diff
@@ -6,55 +6,63 @@
 #define MMSEQS_NCBITAXONOMY_H
 
 #include <map>
+#include <unordered_map>
 #include <vector>
 #include <string>
 
+using TaxID = int;
```

Comment:
Can we do a typedef instead of using?

Reply:
Sure. Have a couple of C++11-isms, the others being the foreach loop as you noted. Martin said that he'll discuss them with you whether you want to have them, but as long as it's not auto he is not opposed.

Reply:
I noticed that we use 0x for the GNU compiler. I am not sure why C++-11 is active for llvm. We should avoid the foreach loops. Sorry for the confusion.

Reply:
Oh ok, not too much to change. Will fix it.